Recent build.gradle changes break CDA run task

The recent merged PR #755 made a change to the build.gradle file, swapping 
`tomcatLibs(libs.bundles.tomcat.embedded)
    tomcatLibs(libs.bundles.tomcat.support)`
for 
` testImplementation(libs.bundles.tomcat.embedded)
    testImplementation(libs.bundles.tomcat.support)`

(Commit: https://github.com/USACE/cwms-data-api/commit/c8c7c42da6706f93ad76a07f642a4f8b2eee7f3b)
    
This appears to be the source of a failure of the Gradle run task within IntelliJ. 
The specific error message is this:
`Error: Unable to initialize main class fixtures.TomcatServer
Caused by: java.lang.NoClassDefFoundError: org/apache/catalina/LifecycleListener
Could not load Logmanager "org.apache.juli.ClassLoaderLogManager"
java.lang.ClassNotFoundException: org.apache.juli.ClassLoaderLogManager
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at java.logging/java.util.logging.LogManager$1.run(LogManager.java:239)
	at java.logging/java.util.logging.LogManager$1.run(LogManager.java:223)
Error: Unable to initialize main class fixtures.TomcatServer

	at java.base/java.security.AccessController.doPrivileged(AccessController.java:319)
	at java.logging/java.util.logging.LogManager.<clinit>(LogManager.java:222)
Caused by: java.lang.NoClassDefFoundError: org/apache/catalina/LifecycleListener`

For now, adding the old lines back to the build file allows for a successful run. 

@MikeNeilson

--- a/cwms-data-api/build.gradle
+++ b/cwms-data-api/build.gradle
@@ -10,6 +10,7 @@ configurations {
     docker
     baseLibs
     tomcatLibs
+    testImplementation.extendsFrom(tomcatLibs)
 }
 
 configurations.implementation {
@@ -137,13 +138,12 @@ dependencies {
     testImplementation(libs.rest.assured)
     testImplementation(libs.hamcrest.all)
 
-    testImplementation(libs.bundles.tomcat.embedded)
-    testImplementation(libs.bundles.tomcat.support)
+    tomcatLibs(libs.bundles.tomcat.embedded)
+    tomcatLibs(libs.bundles.tomcat.support)
     tomcatLibs(libs.google.flogger.api)
     tomcatLibs(libs.google.flogger.backend)
 
     testImplementation(libs.bundles.testcontainers)
-    
 
     testImplementation(libs.apache.commons.csv)
     testImplementation(libs.cwms.tomcat.auth)
@@ -267,7 +267,6 @@ task integrationTests(type: Test) {
     }
     shouldRunAfter test
     classpath += configurations.baseLibs
-    classpath += configurations.tomcatLibs
     // The before all extension will take care of these properties
     systemProperties += project.properties.findAll { k, v -> k.startsWith("RADAR") && !k.startsWith("RADAR_JDBC") }
     systemProperties += project.properties.findAll { k, v -> k.startsWith("CDA") && !k.startsWith("CDA_JDBC") }

